pkg/snapshotter/snapshotter.go

```diff
@@ -73,7 +73,7 @@ func (s *snapshot) CreateSnapshot(ctx context.Context, snapshotName string, volu
 		return "", "", time.Time{}, 0, false, err
 	}
 
-	klog.V(5).Infof("CSI CreateSnapshot: %s driver name [%s] snapshot ID [%s] time stamp [%d] size [%d] readyToUse [%v]", snapshotName, driverName, rsp.Snapshot.SnapshotId, rsp.Snapshot.CreationTime, rsp.Snapshot.SizeBytes, rsp.Snapshot.ReadyToUse)
+	klog.V(5).Infof("CSI CreateSnapshot: %s driver name [%s] snapshot ID [%s] time stamp [%v] size [%d] readyToUse [%v]", snapshotName, driverName, rsp.Snapshot.SnapshotId, rsp.Snapshot.CreationTime, rsp.Snapshot.SizeBytes, rsp.Snapshot.ReadyToUse)
```

Comment:
Can you remove this change?  This is not related to upgrading to v19.

Reply:
I've removed the change.

It was included because unit tests were failing locally without making this modification.

Reply:
Can you submit a separate PR for this?

Reply:
This has been submitted as https://github.com/kubernetes-csi/external-snapshotter/pull/338